docker-compose.yml

```diff
@@ -202,7 +202,7 @@ services:
       timeout: 5s
       retries: 5
 
-  infrahub-server:
```

Comment:
Heuu, last time we changed that we faced many side effects, don't know if we want to do that again

Reply:
you are right